calicoctl/calicoctl.py

```diff
@@ -97,29 +97,32 @@ def keyboard_interrupt_handler(signal, frame):
 
     # Dispatch the appropriate subcommand
     try:
-        command = command_args['<command>']
-
-        # Look for a python file in the calico_ctl module which
-        # shares the same name as the input command
-        command_module = getattr(calico_ctl, command)
-
-        # The command module should have a function the same name as the
-        # command.  This protects against trying to run an invalid command that
-        # happens to have the same name as a non-command module.
-        if not hasattr(command_module, command):
+        try:
```

Comment:
I hit an `AttributeError` in my code and it got caught and suppressed by the except: block that was intended to catch `AttributeError` in the command lookup.  I assumed it should get caught by the `BaseException` except block below.